server/routerlicious/packages/services-utils/src/scheduleJob.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+import * as cron from "node-cron";
+import { ILogger } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+
+export async function scheduleJob<T>(
+    api: () => Promise<T>,
+    config: Provider,
+    featureGateConfigName: string,
+    callName: string,
+    cronExpressionConfigName: string,
+    logger?: ILogger,
+): Promise<any> {
+    let task;
+    if (config.get(featureGateConfigName)) {
```

Comment:
Why do we need to pass the `config` and the `featureGateConfigName` plus `cronExpressionConfigName`, making this function access the config? We could make this file not depend on `nconf` by instead passing the `featureGateConfigName`'s value and the `cronExpressionConfigName`'s value directly. In other words, we would get those configs in the caller who is calling `scheduleJob`.